templates/cinder/config/cinder-backup-config.json

```diff
@@ -7,6 +7,12 @@
       "owner": "root:cinder",
       "perm": "0640"
     },
+    {
+      "source": "/usr/local/bin/container-scripts/kolla_extend_start",
+      "dest": "/usr/local/bin/kolla_extend_start",
+      "owner": "root:cinder",
+      "perm": "0740"
```

Comment:
0740 looks odd. I expect 0640 is all we need if it's sourced by the main kollla script

Reply:
https://github.com/openstack/kolla/blob/master/docker/base/start.sh#L20

Reply:
Yes, these are wrong, thanks for catching it.